odrive_node/README.md

```diff
@@ -50,6 +50,10 @@ For information about installation, prerequisites and getting started, check out
 
   This service requires regular heartbeat messages from the ODrive to determine the procedure result and will block until the procedure completes, with a minimum call time of 1 second.
 
+* `/clear_errors`: Clears disarm_reason and procedure_result and re-arms the brake resistor if applicable
+
+  Change the axis state after clearing the errors, as the axis state is reset to idle when an error occurs.
```

Comment:
This sentence I find a bit confusing. If I understand correctly, you meant something like: "If the axis dropped into IDLE because of an error, clearing the errors does not put the axis back into CLOSED_LOOP_CONTROL. To do so, you must request CLOSED_LOOP_CONTROL again explicitly."

Reply:
That is what I was trying to say. I am sorry that my expression is not clear and thank you for your correction.